### PR TITLE
Setup for docker

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
For deploying to azure

### Changes proposed in this pull request
Add docker config

### Guidance to review
`docker-compose build`
`docker-compose up`

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby File.read(".ruby-version").chomp
+ruby '2.5.3'
```
Comment (benilovj) [inline Gemfile]: why this change? It's still better to set the ruby `.ruby-version` as this is used by Rbenv...

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+version: "3"
+
+services:
+  postgres:
+    image: postgres
```
Comment (benilovj) [inline docker-compose.yaml]: should this pin postgres to the version we're using in production?

File: Gemfile (hunk 1 above)
Comment (dankmitchell) [reply Gemfile]: Docker couldn't read the file. Maybe theres another way to do it...

**Comment by dankmitchell** [comment]
@tvararu do we need to update this docker config?
